app/javascript/flavours/glitch/initial_state.js

```diff
@@ -81,14 +82,19 @@
  * @property {boolean=} use_pending_items
  * @property {string} version
  * @property {boolean} translation_enabled
- * @property {object} local_settings
```

Comment:
Why did you remove `local_settings`?

Reply:
It was applied to the wrong type def.
I moved it to the typedef of InitialState